--- a/protobufs/meshtastic/deviceonly.proto
+++ b/protobufs/meshtastic/deviceonly.proto
@@ -7,14 +7,12 @@ import "meshtastic/mesh.proto";
 import "meshtastic/telemetry.proto";
 import "meshtastic/config.proto";
 import "meshtastic/localonly.proto";
-import "nanopb.proto";
 
 option csharp_namespace = "Meshtastic.Protobufs";
 option go_package = "github.com/meshtastic/go/generated";
 option java_outer_classname = "DeviceOnly";
 option java_package = "com.geeksville.mesh";
 option swift_prefix = "";
-option (nanopb_fileopt).include = "<vector>";
 
 
 /*
@@ -235,7 +233,7 @@ message NodeDatabase {
   /*
    * New lite version of NodeDB to decrease memory footprint
    */
-  repeated NodeInfoLite nodes = 2 [(nanopb).callback_datatype = "std::vector<meshtastic_NodeInfoLite>"];
+  repeated NodeInfoLite nodes = 2;
 }
 
 /*
